webapp/src/components/DynamicButtonDataTable.vue

```diff
@@ -45,23 +45,15 @@
       </button>
     </div>
     <div class="button-right d-flex">
-      <button
-        v-if="itemsSelected.length > 0 && dataType != 'collections'"
-        class="btn btn-default"
-        :disabled="itemsSelected.length === 0"
-        @click="$emit('open-add-to-collection-modal')"
-      >
-        Add to collection
-      </button>
-      <button
+      <SplitButton
```

Comment:
As we discussed, we should probably just use the bootstrap dropdown

Reply:
agree, that's probably a bit lighter weight and more consistent in style (though the primevue one also looks nice)

Reply:
We also couldn't come up with a good label for the button, but perhaps:

- disabling the button when nothing is selected
- labelling it with "Selected..." when nothing is selected
- label it with "n Selected" when n items are selected

Reply:
All done! Let me know if you want to change the label.